src/util/sfdc.ts

```diff
@@ -127,7 +127,13 @@ export const sfdc = {
    * @param url
    */
   isInternalUrl: (url: string): boolean => {
-    const INTERNAL_URL_PARTS = ['.internal.', '.vpod.', 'stm.salesforce.com', '.blitz.salesforce.com'];
+    const INTERNAL_URL_PARTS = [
+      '.internal.',
+      '.vpod.',
+      'stm.salesforce.com',
```

Comment:
I am seeing different URLs for steam this morning on stmfa

on dev hub `https://inttestdevhub02-dev-ed.lightning.stmfa.stm.force.com/`

Reply:
so much fun going on

Reply:
added that, too.